include/gbinder_writer.h

```diff
@@ -119,6 +119,16 @@ gbinder_writer_append_fd(
     GBinderWriter* writer,
     int fd); /* Since 1.0.18 */
 
+gsize
+gbinder_writer_bytes_written(
+    GBinderWriter* self); /* since 1.0.21 */
```

Comment:
`self` -> `writer` here please (and for the next function too). The convention is to use `self` internally in .c files, and give the arguments more descriptive names in .h files